ci-docker/Dockerfile

```diff
@@ -1,13 +1,15 @@
 ARG TARGET_DOCKER_PLATFORM
 
-FROM ubuntu:18.04
+FROM ubuntu:18.10
```

Comment:
Any reason to use newer Ubuntu image? I think we should stick to 18.04 as it's latest LTS release atm.